.ci/openshift-ci/run_smoke_test.sh

```diff
@@ -55,12 +67,11 @@ else
 	info "HTTP server is unreachable"
 fi
 
-info "Deleting resources created"
-oc delete -f ${script_dir}/smoke/service.yaml
-
-# Delete the pod.
+# Delete the resources.
 #
-info "Deleting the ${pod} pod"
```

Comment:
I'd keep the `Deleting the ${pod} pod` message as it pairs with the `Creating the ${pod} pod` one.

Reply:
ack